test/functional/rpc_net.py

```diff
@@ -168,12 +172,111 @@ def test_getnettotals(self):
             self.wait_until(lambda: peer_after()['bytesrecv_per_msg'].get('pong', 0) >= peer_before['bytesrecv_per_msg'].get('pong', 0) + 32, timeout=1)
             self.wait_until(lambda: peer_after()['bytessent_per_msg'].get('ping', 0) >= peer_before['bytessent_per_msg'].get('ping', 0) + 32, timeout=1)
 
+    def test_getnetmsgstats(self):
```

Comment:
I think I might also be able to do some testing by comparing the results of `getnetmsgstats` with that of the `getpeerinfo` RPC? Specifically against the `bytessent_per_msg` and `bytesrecv_per_msg` return objects.